.github/workflows/test.yml

```diff
@@ -68,10 +68,28 @@ jobs:
         if: matrix.platform != 'macos-latest'
         uses: ./.github/actions/free-disk-space
 
-      - name: Install the expect and skopeo package
+      - name: Install the expect
         if: startsWith(matrix.platform, 'ubuntu')
         run: |
-          sudo apt-get install -y expect skopeo
+          sudo apt-get install -y expect
+
+      - name: Install go for skopeo
+        if: startsWith(matrix.platform, 'ubuntu')
+        uses: actions/setup-go@v5
+        with:
+          cache: false
+          go-version: 1.22.x
+
+      - name: install skopeo from source
```

Comment:
added a comment, it's because the latest version of skopeo available on ubuntu is 1.4 which is very old and was running into issues downloading the artifacts properly